integration/helpers/utils.ts

```diff
@@ -39,3 +42,19 @@ export async function tearDown(app: Application) {
     console.error(e);
   }
 }
+
+export async function listHelmRepositories(retries = 0): Promise<string> {
+  if (retries < 5) {
+    try {
+      const { stdout: reposJson } = await promiseExec("helm repo list -o json");
+
+      return reposJson;
+    } catch {
+      await new Promise(r => setTimeout(r, 2000)); // if no repositories, wait for Lens adding bitnami repository
+
+      return await listHelmRepositories((retries + 1));
+    }
+  }
+
+  return "[]";
+}
```

Comment:
I really think that this should return `any[]` or maybe even `{ name: string, url: string}[]`

Reply:
I agree with @Nokel81 , feels weird to return `string` and not array of objects (basically parsed json).

Reply:
Actually it's not parsed json that is returned. Should we return parsed json instead?

Reply:
> Actually it's not parsed json that is returned. Should we return parsed json instead?

I think it would make more sense to return parsed json.

Reply:
changed